src/common/snippets/include/snippets/lowered/pass/propagate_subtensors.hpp

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "snippets/lowered/linear_ir.hpp"
+#include "snippets/lowered/pass/pass.hpp"
+
+namespace ov {
+namespace snippets {
+namespace lowered {
+namespace pass {
+
+class UpdateSubtensors : public pass::RangedPass {
```

Comment:
I'd suggest to add description as well. Could you comment out a little bit at least please?

Reply:
Done, thanks